lib/entropy/dep.py

```diff
@@ -632,6 +632,21 @@ def is_valid_package_tag(tag):
         return False
     return True
 
+def _nat_sort_key(key):
+    """
+    Key generator for natural sorting algorithms.
+
+    @param key: Entropy package tag
+    return: array obtained from the input string with strings parsed as
+        strings and numbers parsed as number with the empty string as
+        first and last element.
+    """
+    def convert(text):
```

Comment:
this can be turned into a lambda and defined outside the function.

_sort_key_convert = lambda text: int(text) if text.isdigit() else text.lower()
?

Up to you.

Reply:
Fixed in 5122204